core/src/Cuda/Kokkos_Cuda_Vectorization.hpp

```diff
@@ -48,7 +48,12 @@
 #ifdef KOKKOS_ENABLE_CUDA
 
 #include <type_traits>
-#include <Cuda/Kokkos_Cuda_Version_9_8_Compatibility.hpp>
+
+#if !defined(KOKKOS_COMPILER_CLANG)
+#define KOKKOS_IMPL_CUDA_MAX_SHFL_SIZEOF sizeof(long long)
+#else
+#define KOKKOS_IMPL_CUDA_MAX_SHFL_SIZEOF sizeof(int)
+#endif
```

Comment:
@crtrott should we just drop this one?
https://github.com/kokkos/kokkos/blob/1fb0c284d458c75370094921d9f202c287502325/core/src/Cuda/Kokkos_Cuda_Vectorization.hpp#L138-L141

Reply:
interesting question. Maybe actually test out whether it makes any impact? I think we probably should use 64bit shuffles when we can.

Reply:
its fine for now. Can we open a follow up issue to explore using 8 byte shuffle?

Reply:
See https://github.com/kokkos/kokkos/issues/4205.